src/main/java/uk/gov/hmcts/reform/sscs/model/single/hearing/HearingDetails.java

```diff
@@ -24,10 +24,10 @@ public class HearingDetails {
 
     private HearingWindow hearingWindow;
 
-    private Number duration;
-
     private List<String> nonStandardHearingDurationReasons;
 
+    private Number duration;
```

Comment:
Do you mind undoing these unessesary changes to this file?

Reply:
Done